### PR TITLE
TF-2907 Disable `YES/NO/MAYBE` action for events when missing `METHOD/ORGANIZER/ATTENDEES` in ics

### PR DESCRIPTION
**Pull request by dab246**
## Issue

#2907 

## Related

- https://github.com/linagora/jmap-dart-client/pull/88

## Demo

https://github.com/linagora/tmail-flutter/assets/80730648/fff8145a-4e81-429c-b960-f6dd61cec47d

### CODE REVIEW

**Comment by dab246** [comment]
Rebase to the `refactor` branch is complete

**Comment by tddang-linagora** [comment]
Tested on Web

**Review by tddang-linagora — APPROVED** [review]

**Comment by hoangdat** [comment]
- [ ] @hoangdat applying change from jmap_dart_client `sprint25_from_migrate_5jun` to this PR

**Comment by hoangdat** [comment]
- [ ] accept response to an error, but front-end still indicate the result
image

**Comment by dab246** [comment]
> * [x]  accept response to an error, but front-end still indicate the result
> 
> image

Fixed. Please test again @hoangdat 


https://github.com/linagora/tmail-flutter/assets/80730648/1b58c68c-6e85-4f73-8225-330428ba50ba

File: lib/features/email/presentation/controller/single_email_controller.dart [hunk 1]
```diff
@@ -1721,6 +1725,28 @@ class SingleEmailController extends BaseController with AppLoaderMixin {
   }
 
   void calendarEventSuccess(CalendarEventReplySuccess success) {
+    final eventReplyResponse = success.calendarEventReplyResponse;
+    log('SingleEmailController::calendarEventSuccess:EventReplyResponse = $eventReplyResponse');
+    if (eventReplyResponse is CalendarEventAcceptResponse
+        && (eventReplyResponse.accepted == null
+            || eventReplyResponse.accepted?.isEmpty == true)) {
```
Comment (hoangdat) [inline lib/features/email/presentation/controller/single_email_controller.dart:1732]: why not handle it in `interactor` or `datasource`?

Comment (dab246) [reply lib/features/email/presentation/controller/single_email_controller.dart]: Sounds reasonable. Fixed.

**Review by hoangdat — APPROVED** [review]

**Review by tddang-linagora — APPROVED** [review]
LGTM

**Pull request merged**